docs/platform/concepts/service-power-cycle.rst

```diff
@@ -0,0 +1,39 @@
+Service power cycle
+===================
+
+Aiven service power off and power on is more than stopping and starting a service on nodes. For better utilisation of resources on Aiven platform, idle resources will be released and only the necessary data will be kept after power off. The impact on the service is different depending on the service type and plan. 
+
+.. warning:: Data loss may happen during a service power off, so it is important for users to understand the consequences before powering off a service.
```

Comment:
I would mention for which services this represents a problem

Reply:
Thanks @ftisiot . I reworded the warning to make it more informative. But can't specify which services may have data loss since it's not as clear as black and white. The purpose of the document is to help users to understand that.